src/lib/MobileSessionDescriptionHandler.js

```diff
@@ -503,9 +503,10 @@ export default SIPMethods =>
             'ICE candidate received: ' + (e.candidate.candidate === null ? null : e.candidate.candidate.trim())
           );
         } else if (e.candidate === null) {
+          // We don't have to use this
           // indicates the end of candidate gathering
-          self.logger.log('ICE candidate gathering complete');
-          self.triggerIceGatheringComplete();
+          // self.logger.log('ICE candidate gathering complete');
```

Comment:
Just making sure you want to leave these commented-out lines in there.

Reply:
Yes as the comment above indicate, we don't want to use it anymore.